Makefile

```diff
@@ -45,6 +45,7 @@ check-node-6:
 
 .PHONY: build-node
 build-node: check-node-6 node-modules
+	sed -i.bak "s|const pjsonVersion = '.*'|const pjsonVersion = '$(shell node -p 'require("./package.json").version')'|" src/tracer.js && rm src/tracer.js.bak
```

Comment:
creating the `.bak` file and removing it is necessary for cross-platform support for in-place edits. Different versions of `sed -i` require a different syntax to specify not to create a backup file. Either `sed -i''` or `sed -i ''`. That space makes all the difference.